Maths.java

```diff
@@ -13,4 +13,13 @@ public static int addtwonumbers(int a, int b) {
     public static int subtracttwonumbers(int a, int b) {
         return a - b;
     }
+
+    public static int productoftwonumbers(int a, int b) {
```

Comment:
the next Pull Request should change function names to use camelCase

e.g. change addtwonumbers to addTwoNumbers.